rakefile.netstd.rb

```diff
@@ -99,7 +100,7 @@
 end

```

Comment:
@blairconrad good spot. Take a look at my latest commit. If OK it should probably be squashed before merge.